CondFormats/CTPPSReadoutObjects/plugins/TotemDAQMappingESSourceXML.cc

```diff
@@ -291,6 +336,22 @@ edm::ESProducts< boost::shared_ptr<TotemDAQMapping>, boost::shared_ptr<TotemAnal
 
 //----------------------------------------------------------------------------------------------------
 
+DOMDocument* TotemDAQMappingESSourceXML::GetDOMDocument(string file)
+{
+  XercesDOMParser* parser = new XercesDOMParser();
```

Comment:
@jan-kaspar - it looks like this misses XMLPlatformUtils::Initialize(); and terminate calls and leaks the parser. is all of that done somewhere else?